app/models/bookmarks.py

```diff
@@ -0,0 +1,16 @@
+from flask import current_app
+from .. import db, login_manager
+
+class Bookmarks(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    listing_id = db.Column(db.Integer, unique=True)
+    merchant_id = db.Column(db.Integer)
+    vendor_id = db.Column(db.Integer, db.ForeignKey('person.id')) #also needs to be added to Hunter's Vendors model
+
+    def __init__(self, listing_id, merchant_id, vendor_id):
+        self.listing_id = listing_id
+        self.merchant_id = merchant_id
+        self.vendor_id = vendor_id
```

Comment:
okay, so my bad. I mixed up merchant and vendor. We still need a one to many relationship between merchants and bookmarks then? So the foreign key is going to be merchant.id? Depending on how @Huntrr implemented it?